src/relion/zocalo/wrapper.py

```diff
@@ -84,6 +84,10 @@ def run(self):
 
             ispyb_command_list = []
 
+            if pathlib.Path(self.params["stop_file"]).is_file():
```

Comment:
```suggestion
            if pathlib.Path(self.params["stop_file"]).is_file():
                relion_prj.load()
```

Sorry, I forgot that it is probably best to load the `Project` again before so that all the information on scheduled jobs is updated